PhysicsTools/NanoAOD/python/custom_run3scouting_cff.py

```diff
@@ -0,0 +1,610 @@
+import FWCore.ParameterSet.Config as cms
+from  PhysicsTools.NanoAOD.common_cff import *
+from PhysicsTools.NanoAOD.simpleCandidateFlatTableProducer_cfi import simpleCandidateFlatTableProducer
+
+def AddRun3Scouting(process):
```

Comment:
following the other comment on configuration, we would rather have this defined as a plain cff, instead of a customise function that is difficult to follow (c.f. https://indico.cern.ch/event/1246376/#1-news)